alacritty/src/window.rs

```diff
@@ -20,16 +20,18 @@ use glutin::dpi::{LogicalPosition, LogicalSize, PhysicalPosition, PhysicalSize};
 use glutin::event_loop::EventLoop;
 #[cfg(target_os = "macos")]
 use glutin::platform::macos::{RequestUserAttentionType, WindowBuilderExtMacOS, WindowExtMacOS};
-#[cfg(not(any(target_os = "macos", target_os = "windows")))]
-use glutin::platform::unix::{WindowBuilderExtUnix, WindowExtUnix, EventLoopWindowTargetExtUnix};
+#[cfg(not(any(target_os = "macos", windows)))]
+use glutin::platform::unix::{EventLoopWindowTargetExtUnix, WindowBuilderExtUnix, WindowExtUnix};
 #[cfg(not(target_os = "macos"))]
 use glutin::window::Icon;
-use glutin::window::{CursorIcon, Fullscreen, Window as GlutinWindow, WindowBuilder};
+use glutin::window::{
+    CursorIcon, Fullscreen, Window as GlutinWindow, WindowBuilder, WindowId as GlutinWindowId,
```

Comment:
Window is only renamed to `GlutinWindow`, because the name `Window` is already taken. I don't think we need to rename the `WindowId`.